MParT/MapFactory.h

```diff
@@ -151,16 +151,16 @@ namespace mpart{
 
         template<typename MemorySpace>
         std::shared_ptr<ConditionalMapBase<MemorySpace>> CreateSigmoidComponent(
-            FixedMultiIndexSet<MemorySpace> mset_offdiag, FixedMultiIndexSet<MemorySpace> mset_diag,
+            FixedMultiIndexSet<MemorySpace> mset_diag,
```

Comment:
Just make this `mset` (`mset_diag` is confusing since there's no "other" kind of `mset`)